shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/restaurant/EmployeeClient.kt

```diff
@@ -0,0 +1,19 @@
+package org.itmodreamteam.myrest.shared.restaurant
+
+import org.itmodreamteam.myrest.shared.error.ClientException
+import kotlin.coroutines.cancellation.CancellationException
+
+interface EmployeeClient {
+
+    @Throws(CancellationException::class, ClientException::class)
+    suspend fun getEmployeesOfRestaurant(restaurantId: Long): List<EmployeeInfo>
+
+    @Throws(CancellationException::class, ClientException::class)
+    suspend fun inviteEmployee(restaurantId: Long, invitation: EmployeeInvitation): EmployeeInfo
```

Comment:
What about removing?

Reply:
We can block employee by updating the status